src/java.base/share/classes/jdk/internal/javac/PreviewFeature.java

```diff
@@ -78,6 +78,8 @@ public enum Feature {
         SCOPED_VALUES,
         @JEP(number=453, title="Structured Concurrency", status="Preview")
         STRUCTURED_CONCURRENCY,
+        @JEP(number=457, title="ClassFile API", status="Preview")
+        CLASSFILE_API,
```

Comment:
Since `Classfile` gets renamed to `ClassFile`, do we rename the title to `Class-File API` and constant to `CLASS_FILE_API`?

Reply:
Rename of classes and interfaces from `Classfile` to `ClassFile` is significant for the API.
`ClassFile` and `Class-File` are considered equal in text.
Internal name of the preview feature is temporary and  invisible in the API.